CMakeLists.txt

```diff
@@ -84,6 +163,8 @@ add_library(kauai)
 file(GLOB kauai-sources CONFIGURE_DEPENDS "${PROJECT_SOURCE_DIR}/kauai/src/*.cpp")
 file(GLOB kauai-headers CONFIGURE_DEPENDS "${PROJECT_SOURCE_DIR}/kauai/src/*.h")
 list(FILTER kauai-sources EXCLUDE REGEX "MAC[.](CPP|cpp)")
+list(FILTER kauai-sources EXCLUDE REGEX "FT[.](CPP|cpp)")
+list(FILTER kauai-sources EXCLUDE REGEX "UT[.](CPP|cpp)")
```

Comment:
```suggestion
list(FILTER kauai-sources EXCLUDE REGEX "(FT|UT)[.](CPP|cpp)")
```

Reply:
Yeah, I'm not a fan of the regexes either... I'll replace them with an explicit list of source filenames.